kratos/utilities/math_utils.h

```diff
@@ -982,6 +982,30 @@ class MathUtils
         c/=norm;
     }
 
+    /**
+     * @brief This computes a orthonormal basis from a given vector
+     * @param a First resulting vector
+     * @param b Second resulting vector
+     * @param c The input vector
+     * @note Orthonormal basis taken from: http://orbit.dtu.dk/files/126824972/onb_frisvad_jgt2012_v2.pdf
+     */
+    template< class T1, class T2 , class T3>
+    static inline void OrthonormalBasis(const T1& c,T2& a,T3& b ){
+        KRATOS_DEBUG_ERROR_IF_NOT(norm_2(c) < 1.0) << "Input should be a normal vector" << std::endl;
```

Comment:
@loumalouomega this check is making the test fail when I compile kratos in debug mode (the input has norm == 1). Are you sure that this is the right check?

Reply:
Huuum, could a problem of tolerance, I can add a less restrict check (BTW if the norm is greater than one will cause errors)

Reply:
well, if this is a problem maybe you need to change the test. The test input is `[0.0, 1.0, 0.0]`.

Reply:
The norm is 1, it is supposed to not give an error